util/mdbook-reggen

```diff
@@ -75,30 +73,6 @@ def main() -> None:
             src_dir,
         )
 
-        def regref_swap(match: re.Match) -> str:
-            """Replaces regref with a link to the register."""
-            reg = match.group(1).split(".")
-            if len(reg) > 3 or len(reg) < 2:
-                sys.exit(
-                    f"{match.group(0)} is invalid. "
-                    "Should be in the form: 'ip_block.register.field', where 'field' is optional.",
-                )
-            try:
-                # Make the path to the config file absolute (to root of the site),
-                # so we don't have to worry about what page we are in.
-                # Also, do the hjson -> html conversion.
-                path = "/{}.html".format(
-                    name2path[reg[0]].removeprefix("./").removesuffix(".hjson")
-                )
-            except KeyError:
-                sys.exit(f"Ip block with name '{reg[0]}' could not be found.")
-
-            name = reg[1] + "." + reg[2] if len(reg) == 3 else reg[1]
-
-            return "[`{}`]({}#{})".format(name, path, reg[1].lower())
-
-        chapter["content"] = REGREF_PATTERN.sub(regref_swap, chapter["content"])
-
```

Comment:
Does this just remove the automatic generation that was needed previously to convert the old to the new docs?

Reply:
This is no longer appears in the history